pkgs/development/python-modules/cocotb-bus/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, cocotb
+}:
+
+buildPythonPackage rec {
+  pname = "cocotb-bus";
+  version = "0.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cc9b0bb00c95061a67f650caf96e3a294bb74ef437124dea456dd9e2a9431854";
+  };
+
+  buildInputs = [ cocotb ];
```

python packages should be propagated
```suggestion
  propagatedBuildInputs = [ cocotb ];
```